src/main/java/touro/peg/Play.java

```diff
@@ -0,0 +1,79 @@
+package touro.peg;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Play {
+    private List<Move> movesList = new ArrayList<>();
+    private List<Move> legalMoves = new ArrayList<>();
+
+    public Play(){
+        populateLegalMoves();
+    }
+
+    public void move(TriangleBoard board, Move move) {
```

Comment:
if all of your methods have a TriangleBoard, then pass it through to the constructor instead of in each method.